OOP시리즈의 마지막 단계로 이번 11단계에서는 아래 두가지 경우에 대해 C++에서 제공하는 try-catch-throw를 이용한 예외처리를 구현한다.
1. 계좌개설 이후, 예금된 금액보다 많은 돈을 출금하려 하는 경우
2. 계좌개설 이후, 0보다 작은 값을 입출금 하려하는 경우

위 두 가지 예외를 처리하기 위해 예외상황 별로 각각 예외 클래스를 정의하고, 사용자에게는 잘못된 입력임을 알리고 올바른 값을 다시 입력하도록 요구한다.

이로 인해 새로 생기는 파일은 Exception.h이다.